kuma/static/js/helpfulness.js

```diff
@@ -42,7 +42,7 @@
         // answers to the simple question include...
         $('#helpful-yes').on('click', function(e) {
             e.preventDefault();
-            confirm('Thanks for your feedback!', 'success', 'Yes');
+            confirm(gettext('Thanks for your feedback!'), 'success', 'Yes');
```

Comment:
The [confirm function](https://github.com/mozilla/kuma/blob/master/kuma/static/js/helpfulness.js#L95) is currently trying to translate these strings, and will need to be modified so that double-translation is not attempted.